### PR TITLE
refactor(traveller): update saas & use standard codes

### PR DESCRIPTION
**Pull request by yusualhashash**
## How has this been tested?

- [ ] Prod
- [ ] Development

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] Added all translations, no hard coded text. 
- [ ] Updated the UI (if needed).
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/page.tsx [hunk 1]
```diff
@@ -1,50 +1,42 @@
...
-  sorting?: string;
-  name?: string;
-  email?: string;
-  fullName?: string;
-  phoneNumber?: string;
-  travelDocumentNumber?: string;
-  username?: string;
-  nationalities?: string;
-  residences?: string;
-  showExpired?: boolean;
```
Comment (ertugrulcan-ays) [inline apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/page.tsx:23]: are we sure all of these unneccesary

**Review by ertugrulcan-ays — CHANGES_REQUESTED** [review]

File: apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/[travellerId]/page.tsx [hunk 2]
```diff
@@ -1,36 +1,35 @@
...
-    lang: params.lang,
+    lang,
   });
 
-  const { languageData } = await getResourceData(params.lang);
-  const traveller = await getTravellersDetailsApi(params.travellerId);
-
-  if (traveller.type !== "success") return notFound();
-  const travellerData = traveller.data;
+  const travellerDetailsData = await getTravellersDetailsApi(travellerId);
```
Comment (eren-ayasofyazilim) [inline apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/[travellerId]/page.tsx:20]: const travellerDetailResponse

File: apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/[travellerId]/identification/[identificationId]/page.tsx [hunk 3]
```diff
@@ -13,30 +13,32 @@ export default async function Page({
...
   await isUnauthorized({
     requiredPolicies: ["TravellerService.Travellers.Edit"],
-    lang: params.lang,
+    lang,
   });
 
-  const { languageData } = await getResourceData(params.lang);
-  const traveller = await getTravellersDetailsApi(params.travellerId);
+  const traveller = await getTravellersDetailsApi(travellerId);
   const countries = await getCountriesApi();
```
Comment (eren-ayasofyazilim) [inline apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/[travellerId]/identification/[identificationId]/page.tsx:23]: const countriesResponse

File: apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/[travellerId]/identification/new/page.tsx [hunk 4]
```diff
@@ -13,13 +13,14 @@ export default async function Page({
...
   await isUnauthorized({
     requiredPolicies: ["TravellerService.Travellers.Create"],
-    lang: params.lang,
+    lang,
   });
 
-  const { languageData } = await getResourceData(params.lang);
-  const traveller = await getTravellersDetailsApi(params.travellerId);
+  const { languageData } = await getResourceData(lang);
+  const traveller = await getTravellersDetailsApi(travellerId);
```
Comment (eren-ayasofyazilim) [inline apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/[travellerId]/identification/new/page.tsx:23]: const travellerResponse

File: apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/[travellerId]/identification/new/page.tsx [hunk 5]
```diff
@@ -13,13 +13,14 @@ export default async function Page({
...
   const countries = await getCountriesApi();
```
Comment (eren-ayasofyazilim) [inline apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/[travellerId]/identification/new/page.tsx:24]: const countriesResponse

**Review by eren-ayasofyazilim — CHANGES_REQUESTED** [review]

File: apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/page.tsx (hunk 1 above)
Comment (yusualhashash) [reply apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/page.tsx:23]: yes already the type that i give contain all them but i taked error from split so i convert the type of nationalities and residences from array to string to solve the lint error

export type GetApiTravellerServiceTravellersData = {
    email?: string;
    fullName?: string;
    maxResultCount?: number;
    nationalities?: Array<(string)>;
    phoneNumber?: string;
    residences?: Array<(string)>;
    showExpired?: boolean;
    skipCount?: number;
    sorting?: string;
    travelDocumentNumber?: string;
    username?: string;
};

File: apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/[travellerId]/page.tsx (hunk 2 above)
Comment (yusualhashash) [reply apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/[travellerId]/page.tsx]: done

File: apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/[travellerId]/identification/new/page.tsx (hunk 4 above)
Comment (yusualhashash) [reply apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/[travellerId]/identification/new/page.tsx]: done

File: apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/[travellerId]/identification/new/page.tsx (hunk 5 above)
Comment (yusualhashash) [reply apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/[travellerId]/identification/new/page.tsx]: done

File: apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/[travellerId]/identification/[identificationId]/page.tsx (hunk 3 above)
Comment (yusualhashash) [reply apps/web/src/app/[lang]/(main)/(unirefund)/parties/travellers/[travellerId]/identification/[identificationId]/page.tsx]: done

**Review by eren-ayasofyazilim — DISMISSED** [review]

**Review by eren-ayasofyazilim — APPROVED** [review]

**Review by ertugrulcan-ays — APPROVED** [review]

**Pull request merged**